frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsListComponent.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import InputModuleProps from 'src/Interface/InputModuleProps';
+import { faPencil, faTrash } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  StyledListContainer,
+  StyledListItem,
+  StyledFontAwesomeContainer,
+  StyledFieldInfoList,
+  StyledCommentContainerDesktop,
+  StyledCommentContainerMobile,
+  StyledDivider,
+} from './FieldsListComponent.styles';
+
+const FieldsListComponent: React.FC<InputModuleProps> = ({ farmDetails }) => (
+  <StyledFieldInfoList>
+    {farmDetails.Fields.slice(1).map((fields) => (
```

Comment:
```suggestion
    {farmDetails.Fields.slice(1).map((fields) => (
```
Mostly nitpicking, but you can type the `fields` variable

Reply:
not sure about what you meant here.

Reply:
```ts
  {farmDetails.Fields.slice(1).map((fields: YourFieldsType) => (
```

Reply:
Oh I see there's confusion because Github thought I was suggesting a change Ahaha, That was an accident

Reply:
Oh yeah hahaha absolutely I can do that. Just to be precise.